js/ctrls/main.js

```diff
@@ -398,6 +398,10 @@ function(
 				animCollapsed: true,
 				files: [],
 			};
+			if (d.verifiedLength)
+				ctx.verifiedLength = d.verifiedLength;
```

Comment:
No, verifiedLength only exists while the download is actively being verified. Similarly verifyIntegrityPending only exists while the download is in the queue to be verified.

Its easiest to notice on peicehashes like torrents since they take a bit longer to hash. Try creating a torrent for something like 10gb of files on your system and then checking integrity on them (Or grabbing your favorite 10gb torrent from somewhere).